### PR TITLE
customization: document templates

### PR DESCRIPTION
**Pull request by fenekku**

### CODE REVIEW

File: docs/quickstart/customize.md [hunk 1]
```diff
@@ -18,3 +20,24 @@ Updating static files...
 Go to the browser [*https://localhost/*](https://localhost) or refresh the page. And voila! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## CSS + Templates
```
Comment (ppanero) [inline docs/quickstart/customize.md]: Moderate: I would remove the CSS part just not to give false hopes. We have a series of issues with webpack, watchpack and invenio-assets (see https://github.com/inveniosoftware/invenio-assets/commit/094c1ef4d0c67b46ab890d8303d51bab1782ba55 and https://github.com/inveniosoftware/invenio-app-rdm/issues/5 for more info)

File: docs/quickstart/customize.md [hunk 2]
```diff
@@ -18,3 +20,24 @@ Updating static files...
...
+
+If you want to customize the look and feel of your instance, you will want to customize the css and templates used.
+
+For the templates, you can just put your new Jinja templates in the `templates/` folder and refer to them from the `invenio.cfg` file appropriately.
```
Comment (ppanero) [inline docs/quickstart/customize.md]: Minor: Shall we link to https://invenio-theme.readthedocs.io/en/latest/configuration.html in order to provide the full list of configuration available?

File: docs/quickstart/customize.md (hunk 1 above)
Comment (fenekku) [reply docs/quickstart/customize.md]: Makes sense. Let's add it back when those issues are solved.

File: docs/quickstart/customize.md (hunk 2 above)
Comment (fenekku) [reply docs/quickstart/customize.md]: Yes, that's more useful. Added.

**Comment by fenekku** [comment]
Merged with changes. Thanks.

**Comment by fenekku** [comment]
Ah this is super old and more recent merged work takes over this. Closing!

**Pull request closed without merging**